### PR TITLE
Some low-impact cppcheck finding fixes.

### PR DESCRIPTION
**Pull request by catenocrypt**
## Description

Fixes for several minor, low-impact cppfindings.  See #922 .

## Testing instructions

Unit tests.

## Types of changes

* Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] Prefix PR title with `[WIP]` if necessary.
- [x] Add tests to cover changes as needed.
- [x] Update documentation as needed.
- [x] If there is a related Issue, mention it in the description (e.g. 'Fixes #444 ).

### CODE REVIEW

File: src/Aeternity/Signer.h [hunk 1]
```diff
@@ -21,14 +21,14 @@ class Signer {
...
+    static Data buildRlpTxRaw(Data& txRaw, Data& sigRaw);
 
-    static Data buildMessageToSign(Data &txRaw);
+    static Data buildMessageToSign(Data& txRaw);
 
     static Proto::SigningOutput createProtoOutput(std::string &signature, const std::string &signedTx);
 
     /// Encode a byte array into base64 with prefix and a checksum
-    static std::string encodeBase64WithChecksum(const std::string &prefix, const TW::Data &rawTx);
+    static std::string encodeBase64WithChecksum(const std::string &prefix, const TW::Data& rawTx);
```
Comment (hewigovens) [inline src/Aeternity/Signer.h:31]:
```suggestion
    static std::string encodeBase64WithChecksum(const std::string& prefix, const TW::Data& rawTx);
```

File: src/Harmony/Signer.cpp [hunk 2]
```diff
@@ -22,13 +22,13 @@ std::tuple<uint256_t, uint256_t, uint256_t> Signer::values(const uint256_t &chai
 }
 
 std::tuple<uint256_t, uint256_t, uint256_t>
-Signer::sign(const uint256_t &chainID, const PrivateKey &privateKey, const Data &hash) noexcept {
+Signer::sign(const uint256_t &chainID, const PrivateKey &privateKey, const Data& hash) noexcept {
```
Comment (hewigovens) [inline src/Harmony/Signer.cpp:25]:
```suggestion
Signer::sign(const uint256_t& chainID, const PrivateKey& privateKey, const Data& hash) noexcept {
```

File: src/Harmony/Signer.cpp [hunk 3]
```diff
@@ -22,13 +22,13 @@ std::tuple<uint256_t, uint256_t, uint256_t> Signer::values(const uint256_t &chai
...
     auto signature = privateKey.sign(hash, TWCurveSECP256k1);
     return values(chainID, signature);
 }
 
 template <typename T>
-Proto::SigningOutput Signer::prepareOutput(const Data &encoded, const T &transaction) noexcept {
+Proto::SigningOutput Signer::prepareOutput(const Data& encoded, const T &transaction) noexcept {
```
Comment (hewigovens) [inline src/Harmony/Signer.cpp:31]:
```suggestion
Proto::SigningOutput Signer::prepareOutput(const Data& encoded, const T& transaction) noexcept {
```

**Comment by hewigovens** [comment]
let's add the script to `tools` folder?

File: src/Aeternity/Signer.h (hunk 1 above)
Comment (catenocrypt) [reply src/Aeternity/Signer.h]: done

**Pull request merged**